### DISCUSSION TITLE
Why "-T" can not output 'stl' file in 2D by "-format" ?

### DISCUSSION THREAD

**Discussion by hubocqjtu** [neperfepx/neper#276] [Q&A] [answered]
Dear all,
Why the neper can not output the "stl" file? 
Code:
$ neper -T -n 1000 -id 1 -morpho "voronoi,aspratio(1.5,0.5)" -transform "rotate(0,0,1,45)" -domain "square(5,5):rotate(0,0,1,-45)" -dim 2 -format "stl"
Result:
![fail stl](https://user-images.githubusercontent.com/87802682/126872332-a58d800f-c234-4b4c-b426-8219530dcd1f.jpg)

**Comment by rquey** [comment, marked as answer]
This is fixed in version `4.1.2-12`.
